lib/private/Teams/TeamManager.php

```diff
@@ -50,8 +54,10 @@ public function getProviders(): array {
 			try {
 				/** @var ITeamResourceProvider $provider */
 				$provider = Server::get($providerRegistration->getService());
+				$this->logger->info('Got team resource provider (id.name): ' . $provider->getId() . '.' .$provider->getName());
```

Comment:
```suggestion
				$this->logger->debug('Got team resource provider (id.name): ' . $provider->getId() . '.' .$provider->getName());
```

Unless there is a good reason to send this to info, it feels like spamming the logs to me?